grakn-engine/src/main/java/ai/grakn/engine/data/RedisWrapper.java

```diff
@@ -65,7 +65,7 @@ public void close() {
         private boolean useSentinel = false;
         private Set<String> uriSet = new HashSet<>();
         private String masterName = null;
-        private int poolSize = 128;
+        private int poolSize = 32;
```

Comment:
Added a comment, this can be configured in the properties under 
redis.pool-size